packages/desktop-client/src/components/settings/index.tsx

```diff
@@ -172,17 +172,13 @@ export function Settings() {
             <Button onPress={closeBudget}>Close Budget</Button>
           </View>
         )}
-
         <About />
-
-        {!Platform.isBrowser && <GlobalSettings />}
```

Comment:
```Platform.isBrowser``` was always true - even on electron. 

It's because we're building the client bundle up using the web config and we don't have specific configs for electron. I'm not sure if this has always been the case, or if some code was deleted at some point. Either way, it hasn't been kept up to date and we should probably avoid using it.